### PR TITLE
added possibility to get app version, app installs and last update

### PR DESCRIPTION
**Pull request by posixpascal**

### CODE REVIEW

**Comment by grantchen** [comment]
I have test this N+1 query cause so much time.

 if don't have N+1 Query only take 5 seconds, but when add it need 33 seconds.

So i suggest add a method to App.rb maybe called `fetch_detail_data`, if you need these new fields data just call it。

**Comment by grantchen** [comment]
1. should delete "puts" debug code
2. have line indent problem in app.rb

**Comment by posixpascal** [comment]
I'll push a new commit to fix it :) sorry for the trouble.

File: lib/google_play_search/app.rb [hunk 1]
```diff
@@ -1,5 +1,30 @@
+require 'open-uri'
 module GooglePlaySearch
   class App
-    attr_accessor :id, :name, :url, :developer, :category, :logo_url, :short_description, :rating, :reviews, :price
+    attr_accessor :id, :name, :url, :developer, :category, :logo_url, :short_description, :rating, :reviews, :price, :version, :installs, :last_updated
```
Comment (simsicon) [inline lib/google_play_search/app.rb]: This line is too long, could be split into two lines.

**Pull request merged**